templates/gulpfile.js

```diff
@@ -222,7 +224,7 @@ gulp.task('karma-conf', ['templates'], function () {
  * Test files
  */
 function testFiles() {
-  return new queue({objectMode: true})
+  return queue({objectMode: true})
```

Comment:
It's just to remove the warning in Gulp 3.9.1

"A constructor name should start with an uppercase letter"